app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+class User < ApplicationRecord
+
+    has_many :events, foreign_key: "creator_id", class_name: "Event"
+    has_many :attendances, foreign_key: "attendee_id"
+    has_many :event_attendance, through: :attendances, source: :attended_event
```

Comment:
I think we forgot to remove this reference to `attended_event` here in the `source`

Another thing, I am still a little confused about this. Maybe you should try giving this the `attended_events` name. And yeah no need for the source.

Reply:
We haven't fixed this line